docs/joss_paper/paper.md

```diff
@@ -82,35 +82,35 @@ The loss functions described in the previous sections are often combined with a
 
-An encoder-decoder architecture, such as an adapted U-Net and the local-net [@hu2018weakly] is used for dense prediction such as direct DDF or DVF. An encoder-only architecture, such as ResNet and VGG, is used to predict the parameters of an affine transformation model, with 12 degrees of freedom.
+An encoder-decoder architecture, such as an adapted U-Net and the local-net [@hu2018weakly], is used for dense prediction such as DDF or DVF. An encoder-only architecture, such as ResNet and VGG, is used to predict the parameters of an affine transformation model, with 12 degrees of freedom.
 
-## Model training
-Registration networks are trained using standard minibatch gradient descent, with each minibatch sampling pairs of moving-fixed images and, when available, moving-and-fixed labels.
+## Model training and prediction
+Registration networks are trained using standard minibatch gradient descent, with each minibatch sampling pairs of moving- and fixed images. Optionally, when available, a pair of moving- and fixed labels.
 
-`DeepReg`, at its first release, provides six predefined dataset loaders for three types of image pairs, unpaired, grouped and paired, each with two dataset loaders depends on whether corresponding labels are avaialble. Random sampling methods are used for unpaired images. For grouped images, group will be sampled before randomly sampling intra-group images for train without biasing to different groups. When multiple labels are avaialble associated with each image, another staged sampling methods are used to draw one pair of labels for each sampled image pair, during each training iteration. Details of the available dataset loaders, how to add cutomised loader and the sampling options are provided in a series of tutorials under the `tutorials` folder.
+`DeepReg` currently provides six predefined _dataset loaders_ for three types of image pairs, paired, unpaired and grouped, each with two dataset loaders depends on whether corresponding labels are avaialble. Random sampling methods are used for unpaired images. For grouped images, group will be sampled before randomly sampling intra-group images - a staged sampling strategy - for training without biasing to different within-group data. When multiple labels are avaialble associated with each image, another staged sampling scheme will draw one pair of labels for each sampled image pair, during each training iteration. Details of the available dataset loaders, how to add cutomised loader and the sampling options are provided in the documentation, including a series of tutorials, under the `doc` folder.
```

Comment:
avaialble -> available (happens twice)
For grouped images, group -> ...images, the group
cutomised -> customised